conanfile.py

```diff
@@ -62,25 +80,33 @@ def build(self):
         cmake.build()
 
     def package(self):
-        self.copy(pattern="COPYING*", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="COPYING*", dst="licenses",
+                  src=self._source_subfolder)
         self.copy("Find*.cmake", ".", ".")
 
         # cmake installs all the files
         cmake = self._configure_cmake()
         cmake.install()
 
     def package_info(self):
-        self.cpp_info.libs = ['mongoc-1.0', 'bson-1.0'] if self.options.shared \
-            else ['mongoc-static-1.0', 'bson-static-1.0']
+        if self.options.shared:
+            self.cpp_info.libs = ['mongoc-1.0', 'bson-1.0']
+        else:
+            self.cpp_info.libs = ['mongoc-static-1.0', 'bson-static-1.0']
+
         self.cpp_info.includedirs = [os.path.join("include", "libmongoc-1.0"),
                                      os.path.join("include", "libbson-1.0")]
 
+        if self.options.icu and not self.options.shared:
```

Comment:
You could put a comment here (in the code) about this change. It's well explained in your PR.